Parsers/OracleWebLogicServer/OracleWebLogicServerEvent.txt

```diff
@@ -0,0 +1,75 @@
+// This function parses Oracle WebLogic Server server.log and access.log
+// Usage Instruction : 
+// Paste below query in log analytics, click on Save button and select as Function from drop down by specifying function name and alias as TomcatEvent.

```

Comment:
Please update the function name and alias name to "OracleWebLogicServerEvent".

Thanks,
Aditi Maheshwari

Reply:
updated